gcn_classic_to_json/notices/GECAM_GND/__init__.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from scipy.stats import norm
+
+from ... import utils
+
+src_class_vals = ["GRB", "SFLARE", "KNOWN_SOURCE", "GENERIC_SOURCE"]
+
+containment_prob = norm().cdf(1) - norm.cdf(-1)
+
+
+def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[20:39]  # Unused. According to docs: '76 bytes for the future'
```

Comment:
These fields are unused by `GECAM_GND`, but some of them are used by `GECAM_FLT`. This line here will defeat the coverage tests for `GECAM_FLT`. Please refactor the common parsing code for both `GECAM_FLT` and `GECAM_GND` into a single function that is reused by both parsers.